src/MDFTextAccessibility.m

```diff
@@ -155,10 +155,31 @@ + (BOOL)textColor:(nonnull UIColor *)textColor
 }
 
 + (BOOL)isLargeForContrastRatios:(nullable UIFont *)font {
+  if (font.pointSize >= 18) {
+    return YES;
+  }
+  if (font.pointSize < 14) {
+    return NO;
+  }
+
   UIFontDescriptor *fontDescriptor = font.fontDescriptor;
-  BOOL isBold =
-      (fontDescriptor.symbolicTraits & UIFontDescriptorTraitBold) == UIFontDescriptorTraitBold;
-  return font.pointSize >= 18 || (isBold && font.pointSize >= 14);
+  if ((fontDescriptor.symbolicTraits & UIFontDescriptorTraitBold) == UIFontDescriptorTraitBold) {
+    return YES;
+  }
+
+  // We treat medium as large for accesibility when larger than 14.
+  NSDictionary *fontTraits = [fontDescriptor objectForKey:UIFontDescriptorTraitsAttribute];
+  NSNumber *fontWeight = fontTraits[UIFontWeightTrait];
+#if defined(__IPHONE_8_2)
+  CGFloat fontWeightMedium = UIFontWeightMedium;
+#else
+  CGFloat fontWeightMedium = 0x3fcd70a3e0000000; // Value gotten from "po UIFontWeightMedium"
```

Comment:
Will we throw a warning on conversion? Should fontWeightMedium be double?
Consider making fontWeightMedium const.
Consider adding a //TODO(issue) to simplify once we are iOS 8.2+.